app.js

```diff
@@ -11,9 +11,16 @@ app.use(express.json());
 
 
 app.get('/api/v1/projects/', async (req, res) => {
+  var name = req.query.name;
```

Comment:
Good research on finding this out.
We'll need to test this on a larger database